src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -167,8 +167,26 @@ the MIT License. See LICENSE in the project root for license information. -->
                     </muxc:NavigationViewItem.Icon>
                 </muxc:NavigationViewItem>
             </muxc:NavigationView.PaneFooter>
-            <Frame x:Name="contentFrame"
-                   Margin="20"></Frame>
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="*"></RowDefinition>
+                    <RowDefinition Height="Auto"></RowDefinition>
+                </Grid.RowDefinitions>
+                <Frame x:Name="contentFrame"
+                       Margin="20"
+                       Grid.Row="0"></Frame>
+                <Grid Height="100" Grid.Row="1">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*"></ColumnDefinition>
+                        <ColumnDefinition Width="200"></ColumnDefinition>
+                    </Grid.ColumnDefinitions>
+                    <TextBlock x:Uid="Settings_UnsavedSettingsWarning" Grid.Column="0" Foreground="Goldenrod" FontSize="15" VerticalAlignment="Center" HorizontalAlignment="Left" TextAlignment="Left" Margin="30,0,0,0"/>
```

Comment:
Do we need tooltips for these controls (TextBlock, buttons)?

Reply:
I dunno. The text block is its own description.

Should we simply _hide the entire save/reset/warning panel_ when there's nothing to show?

Reply:
😆 I now realize asking about a text block tooltip is uh a little redundant, I think the `reset` button could benefit from a tooltip clarification though (I can see myself thinking it would "reset to default settings"), or maybe it could be called "Discard Changes"?

Reply:
Probably for the buttons, the textblock is self-explanatory I'd say.